src/main/java/org/generationcp/middleware/liquibase/UpdatePresetsTask.java

```diff
@@ -58,17 +60,26 @@ public void execute(final Database database) throws CustomChangeException {
 					final LabelPrintingPresetDTO labelPrintingPresetDTO =
 						this.jacksonMapper.readValue(configuration, LabelPrintingPresetDTO.class);
 					final List<List<String>> selectedFields = new ArrayList<>();
+					final Statement finalSelectStatement = selectStatement;
 					labelPrintingPresetDTO.getSelectedFields().stream().forEach(list -> {
 						selectedFields.add(list.stream().map((fieldId) -> {
-							return this.concatenateFieldTypeName(fieldId);
+							try {
+								return this.concatenateFieldTypeName(fieldId);
+							} catch (Exception e) {
+								throw new RuntimeException(e);
```

Comment:
@cuenyad I dont think we should halt the process due to a unmatched variable id, we can just skip the value and continue with the processing

Reply:
done!